getting-started/scaling-out.md

```diff
@@ -30,72 +29,65 @@ and removed using [`delete_data_node`][delete_data_node].
-  existing PostgreSQL instance.
+* A data node is added as a local object on the access node and
+  contains the configuration needed to connect to a database on a
+  PostgreSQL instance.
 
-* You should already have a running PostgreSQL server on the data node host.
+* You should already have a running PostgreSQL server on an instance
+  that will host a data node database. The data node's database will
+  be created when executing the `add_data_node` command on the access
+  node and should _not_ exist prior to adding the data node.
 
-* Ensure that the data node has password authentication enabled
-  in their `pg_hba.conf` files for any non-superusers.
+* PostgreSQL instances that will act as data nodes are assumed to
+  contain the same roles and permissions as the access node instance
+  (or, at a minimum, the roles and permissions associated with
+  distributed hypertables). Currently, such roles and permissions need
+  to be created manually, although there is a [utility
+  command][distributed_exec] that can be used to create roles and
+  permissions across all (or a subset of) data nodes.
 
-When creating the data node, you should:
 
-* Provide a name to use when referring to the data node from
-  this database.
+When creating the data node, you should:
 
-* Provide the host where the hypertable partition for the distributed
-  hypertable should be stored.
+* Run `add_data_node` as a superuser.
 
-* Provide the remote password, which will be used by the current user
-  during access to the created remote data node.
+* Ensure that the access node's superuser has a way to
+  [authenticate][data-node-authentication] with the data node
+  instance. This can be done either by setting up password or
+  certificate authentication. Other ways of authenticating may be
+  supported in the future.
 
-* Provide a bootstrap user and password, which is used to
-  create the data node. If the current user can be used, then
-  the boostrap user and password can be omitted.
+* Provide a name to use when referring to the data node from
+  the access node database.
 
-* Ensure that the bootstrap user used for connecting to the data node
-  is a superuser.
-  This is necessary since
-  `add_data_node` expects to be able to create a
-  database on the remote data node and create
-  a TimescaleDB extension within it.
+* Provide the host name (and optionally port) of the PostgreSQL
+  instance that holds data node.
 
-* Ensure that the local user has `USAGE` privileges on the `timescaledb_fdw`
-  foreign data wrapper on the access node.
+After creating the data node:
 
-```sql
-SELECT add_data_node('node1', host => 'dn1.example.com',
-  password=>'<remote_password>', bootstrap_user=>'<superuser>',
-  bootstrap_password=>'<superuser_password>');
+* Ensure that non-superusers have `USAGE` privileges on the
+  `timescaledb_fdw` foreign data wrapper and any
+  data node objects they will use on the access node.
```

Comment:
Hmmm.... maybe something we might consider to do automatically when adding the data node. No action here though.